src/ejabberd_receiver.erl

```diff
@@ -123,6 +124,7 @@ init([Socket, SockMod, Shaper, MaxStanzaSize]) ->
                 sock_mod = SockMod,
                 shaper_state = ShaperState,
                 max_stanza_size = MaxStanzaSize,
+                stanza_chunk_size = 0,
```

Comment:
it should be set to 0, each time we reset stream.

Reply:
Thanks! Let me know if c107deb is not what you had in mind.